astropy/io/votable/tests/converter_test.py

```diff
@@ -48,12 +47,11 @@ def test_char_mask():
 
 def test_oversize_unicode():
     config = {'verify': 'exception'}
-    with catch_warnings(exceptions.W46) as w:
+    with pytest.warns(exceptions.W46) as w:
         field = tree.Field(
             None, name='c2', datatype='unicodeChar',
-            config=config)
+            arraysize='1', config=config)
```

Comment:
@mhvk , so `pytest.raises` was a typo on my part. Nice catch! Also I think the more correct solution is to provide `arraysize` to get rid of the W47 warning that this test is not looking for anyway.

Reply:
Ah, that makes sense. Resolved!